tools/api-markdown-documenter/src/test/HtmlEndToEnd.test.ts

```diff
@@ -3,141 +3,104 @@
  * Licensed under the MIT License.
  */
 
-import * as Path from "node:path";
-import { fileURLToPath } from "node:url";
+import Path from "node:path";
 
-import { ApiItemKind, ReleaseTag } from "@microsoft/api-extractor-model";
-import { FileSystem, NewlineKind } from "@rushstack/node-core-library";
+import { ReleaseTag, type ApiModel } from "@microsoft/api-extractor-model";
 
-import type { DocumentNode } from "../documentation-domain/index.js";
-import {
-	type RenderDocumentAsHtmlConfiguration,
-	renderDocumentAsHtml,
-} from "../renderers/index.js";
+import { HtmlRenderer, loadModel } from "../index.js";
```

Comment:
Both HTML and Markdown end-to-end test suites have been updated to leverage the public API, rather than using internal implementation details.